pybleau/app/ui/dataframe_plot_manager_view.py

```diff
@@ -223,9 +223,10 @@ def _new_plot_button_fired(self):
             return
 
         plot_type = selector.plot_type
-
+        initial_creation = True
```

Comment:
Move this inside the `else` clause since the `if` clause resets it to False?

Reply:
It only sets it to false it if is going to create a plot that is a template.

> Should we move this inside the `else` clause since the `if` clause resets it to False?